pipelines/rj_segur__forca_municipal/task.py

```
@@ -0,0 +1,406 @@
+                timeout=self.timeout
+            ) as client:
+                print(f"Autenticando em {login_url} ...")
+                response = client.post(login_url, json=credentials)
+                response.raise_for_status()
+
+                data = response.json()
+                self.access_token = data.get('accessToken')
+                self.expiration_time = data.get('expirationTime')
+
+                if self.access_token:
+                    print(f"✓ Autenticação bem-sucedida")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)